e2e-cypress/files/cypress.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'cypress'
+import setupNodeEvents from './plugins/index.js'
+export default defineConfig({
+  reporter: 'junit',
+  reporterOptions: {
+    mochaFile: 'build/test-results/tests.xml',
+    toConsole: true,
+  },
+  e2e: {
+    baseUrl: 'Introduce your app URL',
+    fixturesFolder: "fixtures",
+    specPattern: 'tests/**/*.cy.ts',
+    supportFile: "support/e2e.ts",
+    viewportWidth: 1376,
+    viewportHeight: 660,
+    experimentalModifyObstructiveThirdPartyCode:true,
```

Comment:
```suggestion
    experimentalModifyObstructiveThirdPartyCode: true,
```

Reply:
There are some formatting issues here and there, e.g. inconsistant usage of `'foo'` vs. `"foo"`. Have you thought about running some code formatter like [prettier](https://prettier.io)?